src/pegout/components/PegoutForm.vue

```diff
@@ -294,6 +294,14 @@ export default defineComponent({
       showTxErrorDialog.value = true;
     }
 
+    const valueToReceive = computed<WeiBig>(() => {
+      const quoteHash = selectedQuoteHash.value || '';
+      if (quoteHash) {
+        return flyoverPegoutState.value.amountToTransfer;
+      }
+      return pegOutTxState.value.amountToTransfer;
```

Comment:
We have to take into account the fees here

Reply:
I just updated the value on the nativeQuote 👌